guides/v2.1/cloud/cdn/cloud-fastly-custom-response.md

```diff
@@ -0,0 +1,159 @@
+---
+group: cloud-guide
+title: Customize error response pages
+redirect_from:
+   - /guides/v2.1/cloud/configure/fastly-vcl-extend-timeout.html
+   - /guides/v2.2/cloud/configure/fastly-vcl-extend-timeout.html
+   - /guides/v2.3/cloud/configure/fastly-vcl-extend-timeout.html
+   - /guides/v2.1/cloud/access-acct/fastly.html
+   - /guides/v2.2/cloud/access-acct/fastly.html
+   - /guides/v2.3/cloud/access-acct/fastly.html
+   - /guides/v2.1/cloud/cdn/configure-fastly.html#fastly-errpg
+   - /guides/v2.2/cloud/cdn/configure-fastly.html#fastly-errpg
+   - /guides/v2.3/cloud/cdn/configure-fastly.html#fastly-errpg
+
+functional_areas:
+  - Cloud
+  - Setup
+  - Configuration
+---
+
+When a request to the Fastly origin fails, Fastly returns default response pages with basic formatting and generic messaging that can be confusing for users. For example, Fastly returns the following default error page when an origin request fails with a 503.
+
+![Fastly default error page]
+
+You can update your {{ site.var.data.ee }} store configuration to replace some default response pages with pages that have friendlier messaging and improved HTML styling as shown in the following example.
+
+![Fastly custom error page]
+
+Currently, you can customize the following Fastly response pages for your {{ site.data.var.ece }} project through the Magento Admin UI.
+
+- [Timeout or site maintenance outages (503 Service Unavailable)](#customize-the-503-response-page)
+- [WAF blocking events that occur when the WAF detects suspicious request traffic (403 Forbidden)](#customize-the-waf-response-page)
+
+
+**HTML coding requirements**
+
+The HTML code for the custom page must meet the following requirements:
+
+- Content can contain up to 65,535 characters.
+- Specify all CSS inline in the HTML source.
+- Bundle images in the HTML page using base64 so that they display even if Fastly is offline. See [Data URIs on the css-tricks site](https://css-tricks.com/data-uris/).
+
+
+## Customize the 503 response page
+
+Customers see the default 503 response page in the following cases:
+
+- When a request to the Fastly origin returns a response status great than 500.
+- When the Fastly origin is down, for example due to a timeout, maintenance activity, or health issues.
+
+You can customize the default page by adapting the following HTML code to include styling to match your {{ site.data.var.ee }} store theme and modifying the title and messaging as needed.
+
+```html
+<!DOCTYPE html>
+<html>
+   <head>
+      <meta charset="UTF-8">
+         <title>503</title>
+   </head>
+   <body>
+      <p>Service unavailable</p>
+   </body></html>
+```
+
+Verify that the modified source displays correctly in the browser. Then, add the customized HTML code to the Fastly configuration.
+
+### Add the custom response page to the Fastly configuration
```

Comment:
This may be something we need to add to the style guide. In order to differentiate a normal content heading with a procedure heading (so users can identify quickly and jump to the procedure) we have to employ a uniform look. Procedure titles in the Cloud guide will be heading 4, open with a "To do blah blah...", and end with a colon.

`#### To add a custom response page:`

Also, using a heading 4 keeps procedure titles out of the Right Navigation menu, since they are not always short.